qt/python/instrumentview/instrumentview/FullInstrumentViewWindow.py

```
@@ -478,6 +543,42 @@ def add_pickable_main_mesh(self, point_cloud: PolyData, scalars: np.ndarray | st
             render_points_as_spheres=True,
         )
 
+    def add_masked_mesh(self, mesh: PolyData) -> None:
+        if mesh.number_of_points == 0:
+            return
+        # RGB for dark grey is (64, 64, 64), normalised is (0.25, 0.25, 0.25)
+        self.main_plotter.add_mesh(mesh, color=(0.25, 0.25, 0.25), pickable=False, render_points_as_spheres=True, point_size=15)
+
+    def add_cylinder_widget(self, bounds, callback) -> None:
```

Ruff (0.14.6)

Unused method argument: `callback`

(ARG002)